.github/workflows/release.yaml

```diff
@@ -0,0 +1,29 @@
+name: Releases
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
what does this line mean?

Reply:
It uses the checkout action by GitHub for going inside the repo and checking it - https://github.com/actions/checkout

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix